cmd/mesh/upgrade.go

```diff
@@ -0,0 +1,364 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mesh
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/ghodss/yaml"
+	goversion "github.com/hashicorp/go-version"
+	"github.com/spf13/cobra"
+
+	"istio.io/operator/pkg/compare"
+	"istio.io/operator/pkg/manifest"
+	opversion "istio.io/operator/version"
+)
+
+const (
+	// The maximum duration the command will wait until the apply deployment reaches a ready state
+	upgradeWaitSecWhenApply time.Duration = 300
```

Comment:
this is 300 ns, change to:
upgradeWaitSecWhenApply = 300 * time.Second
also below

Reply:
fixed, I  misunderstood of this.